infra/frontend/app-config/env-config/outputs.tf

```diff
@@ -22,3 +22,7 @@ output "incident_management_service_integration" {
     integration_url_param_name = "/monitoring/${var.app_name}/${var.environment}/incident-management-integration-url"
   } : null
 }
+
+output "domain" {
+  value = var.domain
+}
```

Comment:
I'm not saying this is necessary, I think we talked about this last time, but the convention from the service layer has to not use `module.env_config` from the `service/main.tf` file, but rather `module.app_config.environment_configs[var.environment_name]` because the `app_config` exports all of the environment specific configs associated with that app. I'm okay diverging from that convention, but I don't know why it was implemented that way historically.